### PR TITLE
Feature/dropdown component

### PR DESCRIPTION
**Pull request by Stefuu**
Creating dropdown component

## Changes

- creating dropdown option component
- creating dropdown component
- creating story on storybook

## Checklist

- [x] Unit tests
- [x] Documentation

## References *(optional)*

Fixes 
Connects 

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

File: src/components/DropdownOption/DropdownOption.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    title,
+    className,
+    leftIconName,
+    leftIconType = 'outline',
+    rightIconName,
+    rightIconType = 'outline',
+}) => {
+    const cx = require('classnames');
+    
+    const classes = cx(className, 'flex items-center cursor-pointer px-8 h-10 text-gray-800 hover:text-white hover:bg-primary-light hover:fill-white')
```
Comment (joshLong145) [inline src/components/DropdownOption/DropdownOption.tsx:15]: Will using a flexbox internally here cause issues with user layouts?

Comment (Stefuu) [reply src/components/DropdownOption/DropdownOption.tsx:15]: Hum, i don't think so, flexbox will only affect it's children

Comment (Stefuu) [reply src/components/DropdownOption/DropdownOption.tsx:15]: We have flexbox in most of our components, like Button
[nebula-react/src/components/Button/Button.tsx](https://github.com/sonr-io/nebula-react/blob/master/src/components/Button/Button.tsx)

**Review by tgfassina — APPROVED** [review]

File: src/components/DropdownOption/DropdownOption.tsx (hunk 1 above)
Comment (joshLong145) [reply src/components/DropdownOption/DropdownOption.tsx:15]: okay, we can keep the internal flex box.

**Review by joshLong145 — APPROVED** [review]

**Pull request merged**